lib/main.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+require('string.prototype.startswith');
```

Comment:
It's a polyfill. https://github.com/mathiasbynens/String.prototype.startsWith/blob/master/startswith.js

With es6 you get `startsWith` for free.

Would you still suggest a static method?